### DISCUSSION TITLE
EvalEx for Python

### DISCUSSION THREAD

**Discussion by khansaad** [ezylang/EvalEx#286] [Ideas]
Hi Team,

I came across this library when I was searching for an algebraic parsing mechanism for one of the module of my project. It's pretty handy and did the job well as per our requirements. 

Now, there's a need for something similar for a python module. We're thinking of having something within python instead of getting output from this java service through REST. Is there something already available which I'm missing or do you guys have any plan to implement this in Python in coming future?

**Comment by uklimaschewski** [comment]
Hi, I have just started creating version 3.0 of EvalEx in Java and for myself, I have no plan to do a port to Python.
But maybe, there are others who already have a port, are interested in porting or know of a similar library for Python.

**Reply by khansaad** [reply]
@uklimaschewski Ok. Thank you for the quick response. 👍🏽
